src/odemis/driver/lakeshore.py

```diff
@@ -287,32 +355,32 @@ def GetIdentifier(self):
             raise IOError("Invalid identifier received")
 
         return manufacturer, md, serialn, firmware
-    
+
     def GetTemp(self):
         """
         Get the temperature at the thermocouple junction
         """
         return float(self._sendQuery(b"TEMP?"))
 
-    def SetSetpoint(self, temp):
+    def SetSetpoint(self, output_channel, temp):
         """
         Set the temperature setpoint
         temp (float): the temperature to set, in Celsius
```

Comment:
Please update the docstrings of all these functions where you've added a argument, with the description of the argument.